pkgs/development/compilers/gambit/unstable.nix

```
@@ -1,12 +1,13 @@
-{ stdenv, callPackage, fetchgit }:
+{ stdenv, callPackage, fetchFromGitHub }:
 
 callPackage ./build.nix {
-  version = "unstable-2018-09-03";
-# git-version = "4.9.0";
-  SRC = fetchgit {
-    url = "https://github.com/feeley/gambit.git";
-    rev = "7cdc7e7b9194b2c088c0667efaf7686a4ffd0d8a";
-    sha256 = "06mmi8jkinihfirz4gjfw2lhxhskiqf3d47sihzx10r60asyqcxg";
+  version = "unstable-2018-11-16";
+# git-version = "4.9.1";
+  SRC = fetchFromGitHub {
```

this also needs to be adapted in build.nix:

```suggestion
  src = fetchFromGitHub {
```

---

The reason I use `SRC` instead of `src` is that I failed to find the proper syntax in `build.nix` for the `std.mkDerivation` to use the `src` from the environment. I have:
```
{ stdenv, …, SRC } :
std.mkDerivation rec {
   src = SRC;
…}
```
If I tried to use `src = src;` then the `rec` understands that as an attempt to circularly define `src` as itself.

---

`inherit src;`

---

Thanks! I will update the PR after testing is complete.